implementations/rust/ockam/ockam_command/src/node/create/demo_config_files/2.kafka.single-machine.yaml

```diff
@@ -0,0 +1,8 @@
+name: kafka-single-machine
```

Comment:
Maybe `kafka-portal` or `kafka-node` would be a more descriptive name for the node?